lab_2_retrieval_w_bm25/main.py

```diff
@@ -92,6 +163,20 @@ def calculate_tf_idf(tf: dict[str, float], idf: dict[str, float]) -> dict[str, f
 
     In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(tf, dict) or any(
```

Comment:
join checks. The same for other functions

Reply:
done